spec/rubocop/cop/rails/pluralization_grammar_spec.rb

```diff
@@ -0,0 +1,92 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Rails::PluralizationGrammar do
+  subject(:cop) { described_class.new }
+  before(:each) do
+    inspect_source(cop, source)
+  end
+
+  shared_examples 'enforces pluralization grammar' do |method_name|
+    context "When #{method_name} is called on an unknown variable" do
+      context "when using the plural form ##{method_name}s" do
+        let(:source) { "some_variable.#{method_name}s" }
+        it 'does not register an offense' do
+          expect(cop.offenses).to be_empty
+        end
+      end
+
+      context "when using the singular form ##{method_name}" do
+        let(:source) { "some_method.#{method_name}" }
+
+        it 'registers an offense' do
+          expect(cop.offenses).to be_empty
```

Comment:
Yeah, I agree it's a contrived example :sweat_smile:. But I'm still not sure why RuboCop should have an opinion about the pluralization of unknown variables/methods. I suppose because when you have an unknown value, it's very likely to be pluralized?